src/lightning_utilities/core/random.py

```diff
@@ -0,0 +1,17 @@
+import random
+
+from lightning_utilities.core.imports import package_available
+
+
+def seed_all(seed):
```

Comment:
I disagree with adding this. This package is for python utilities. These dependencies are for the lite/pl packages, hence why they should be the ones defining this (and they are!)

Reply:
well, any compute package not only Lite/PL but fine with drop...